### PR TITLE
Tomster player

### PR DESCRIPTION
**Pull request by patsy-issa**
This PR is to help review the changes before force pushing them to the branch, it will remain in draft and will not be merged.

Changes introduced:

- Use template colocation for components.

### CODE REVIEW

File: tomster-player/app/router.js [hunk 1]
```diff
@@ -8,6 +8,6 @@ export default class Router extends EmberRouter {
 
 Router.map(function() {
   this.route('albums', { path: 'library' }, function() {
-    this.route('album', { path: ':album_id'})
+    this.route('album', { path: ':album_id'});
```
Comment (marcoow) [inline tomster-player/app/router.js:12]: Not really related to this PR but we should move this change into the routing commit to avoid confusion.

**Review by marcoow — COMMENTED** [review]
Looks good 👍 

I just recognized 1 (unrelated) thing.

File: tomster-player/app/router.js (hunk 1 above)
Comment (patsy-issa) [reply tomster-player/app/router.js]: Moved the changes to the routing commit

**Review by marcoow — APPROVED** [review]

**Comment by patsy-issa** [comment]
fwiw I forced pushed did not merge

**Pull request merged**